src/components/greeting.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { useEffect } from 'react';
```

Comment:
- [ ] As I mentioned in the previous comment, kindly make sure you fix your linters error instead of disabling them. This is for best practices reasons and clean code